ruby-challenges.rb

```diff
@@ -19,13 +29,47 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+# create metod with nums_array1
+# add all of the #'s together'
+# output the sum of all the nums added
+
+
+def add (newArray)
+  let newArray = []
+  return array.map((value, index) => 
+  if (index > 0) 
+  return newArray = value + newArray
+  else 
+  return newArray = value
+end
+p add()
```

Comment:
This is close but its a few steps off. For one ruby doesn't need the word "return".